cdc/redo/manager.go

```diff
@@ -360,8 +360,11 @@ func (m *ManagerImpl) updateTableResolvedTs(ctx context.Context) error {
 		return err
 	}
 	minResolvedTs := uint64(math.MaxUint64)
-	for tableID, rts := range rtsMap {
-		m.rtsMap[tableID] = rts
+	for tableID := range m.rtsMap {
+		if rts, ok := rtsMap[tableID]; ok {
```

Comment:
why part of tables are not maintained in redo log writer?

Reply:
If there is no data of this table, and flush resolved ts of this table has not been executed, the redo log writer doesn't know this table.
The redo log writer only records the information of the table that it has met.